src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/sdl_pending_resumption_handler.h

```diff
@@ -55,24 +55,29 @@ class SDLPendingResumptionHandler
   void ClearPendingResumptionRequests() OVERRIDE;
 
  private:
+  void RaiseFakeSuccfullResponse(smart_objects::SmartObject response,
```

Comment:
As for me, name of this method is misleading and unclear, even if I've already read this function in source file.

Reply:
Also typo RaiseFakeSuccessfullResponse

Reply:
fixed 3bbc0f34445a0db922a2f167f72c637d238dfcb8